### DISCUSSION TITLE
FIX nativewind package. WIndows OS error. Critical Error.

### DISCUSSION THREAD

**Discussion by 2-fly-4-ai** [t3-oss/create-t3-turbo#1014] [Ideas]
### Describe the feature you'd like to request

I would like to ask the developers behind Native Wind to please escape white space properly in their CLI commands.

This is a critical error that only manifests with Windows users who happen to have a space somewhere in their pathname.

The spaces can exist in either the username or filename causing it to fail.

### Describe the solution you'd like to see

Escape the whitespace properly. 

fix node module nativewind > dist > Metro > tailwind-cli.js

### Additional information

Do something like this:

    const cliCommandPath = options.cliCommand;
    const [nodeExecutable, ...cliPathParts] = cliCommandPath.split(" ");
    const fixedCliCommand = [nodeExecutable, `"${cliPathParts.join(" ")}"`].join(" ");
    
    // Now use fixedCliCommand in your spawnCommands array
    const spawnCommands = [
        fixedCliCommand,
        "--input",
        `"${input}"`,
        "--output",
        `"${options.output}"`,
    ];

**Comment by juliusmarminge** [comment]
Why are you opening this here? Open it at nativewind
